gateway/ln-gateway/src/lib.rs

```diff
@@ -118,6 +118,7 @@ const DEFAULT_MODULE_KINDS: [(ModuleInstanceId, &ModuleKind); 2] = [
 ];
 
 #[derive(Parser)]
+#[command(version)]
```

Comment:
While at it - worth checking if all other binaries have it.

Reply:
Good idea 👍 

https://github.com/fedimint/fedimint/pull/4723